server/lib/pbench/__init__.py

```diff
@@ -152,7 +152,7 @@ def get_pbench_logger(caller, config):
 
     pbench_logger = logging.getLogger(caller)
     if caller not in _handlers:
-        pbench_logger.setLevel(logging.DEBUG)
+        pbench_logger.setLevel(config.logging_level)
```

Comment:
I didn't think this would work but I read the setLevel() documentation and it seems to be fine. I presume you have tested it and it works, right?

Reply:
Yes I have tested it and it is working